docs/articles/actors/routers.md

```diff
@@ -96,6 +96,27 @@ Routers are implemented as actors, so a router is supervised by it's parent, and
 
 By default, pool routers use a custom strategy that only returns `Escalate` for all exceptions, the router supervising the failing worker will then escalate to it's own parent, if the parent of the router decides to restart the router, all the pool workers will also be recreated as a result of this.
 
+## Sending messages to actors behind router
+
+As routers are actors, we send over messages to actors they manage the same way we communicate to other actors via `ActorRef`:
```

Comment:
Same problem as above. It would be better if it said: "As routers are actors, they communicate with their `Routees` the same way normal `Actors` would communicate with each other.